README.md

```diff
@@ -101,6 +101,45 @@ scripts.
 If you would like to contribute your own demo, see our [demo submission
 guide](https://pennylane.ai/qml/demos_submission.html).
 
+## Build using Docker
+
+
+Docker support exists for building using CPU and GPU (Nvidia CUDA 11.1+) images.
+
+**Note: Docker builds using "make" will work on Linux and MAC only, For MS Windows one can use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)  
+
+To build a basic PennyLane image without any additional interfaces (Torch, TensorFlow, or Jax) run following:
+
+```
+make -f docker/docker.makefile build-base

```

Comment:
This fails to run with error: `docker/docker.makefile:19: *** missing separator.  Stop.` and also prevents all the other images from currently being built and tested.